unitycatalog-ai/integrations/crewai/pyproject.toml

```
@@ -0,0 +1,68 @@
+[project]
+name = "ucai-crewai"
+version = "0.0.1"
```

```suggestion
version = "0.1.0"
```
Preparation for release :)